RELEASE.md

```diff
@@ -32,7 +52,8 @@ Please follow the established format:
 - Add web worker to make the expensive graph layout calculation into an asynchronous action, to prevent it from blocking other tasks on the main thread. (#217)
 - Focus search bar with Cmd+F/Ctrl+F keyboard shortcuts (#261)
 - Allow an argument to be passed to loadJsonData, for external use if needed (#215)
-- Add support for multiple pipelines. This is a work-in-progress, and is currently disabled by default and hidden behind a flag. (#192, #215, #216, #221, #252, #254)
+- Add support for multiple pipelines. This is a work-in-progress, and is currently disabled by default and hidden behind a flag. (#192, #215, #216, #221, #254)
+- Begin adding individual node API endpoints, as a prelinary step towards full node metadata sidebars (#252)
```

Comment:
Was this missing from the last release?

Reply:
I had included `#252` under the wrong bullet point, so I've fixed it. It wasn't really part of the multiple pipelines feature, it was part of the metadata panel feature